### PR TITLE
CircleCI status badge added

### PR DESCRIPTION
**Pull request by NexFlare**
Removed travis status badge and added circleci status badge

### CODE REVIEW

**Review by kontaras — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -2,34 +2,35 @@
...
+- Time Series database — for storage of real-time, high volume metrics (e.g. InfluxDB, Prometheus, LogStash)
+- Query & Visualization — that enables visual monitoring and root cause analysis (e.g. Grafana)
 
 NuoDB Collector utilizes a popular open-source collector - `telegraf`.
 It's designed to be used alongside a NuoDB engine process to collect metrics from the engine and publish those metrics to a time series database.
 
 Built into this container are 4 input plugins to collect metrics from the NuoDB engine:
 
-1.  `metrics` - collects the [Engine Metrics](https://doc.nuodb.com/nuodb/4.0.x/reference-information/metrics-published-by-database-processes/)  on a
+1.  `metrics` - collects the [Engine Metrics](https://doc.nuodb.com/nuodb/4.0.x/reference-information/metrics-published-by-database-processes/) on a
```
Comment (adriansuarez) [inline README.md:21]: This is a dead link. Can you change it to https://doc.nuodb.com/nuodb/latest/reference-information/metrics-published-by-database-processes ?

File: README.md [hunk 2]
```diff
@@ -2,34 +2,35 @@
...
 
 Most modern application monitoring systems consist of the following 3
 core components:
 
-* Collector             — daemon(s) to gather metrics such as this repository
-* Time Series database  — for storage of real-time, high volume metrics (e.g. InfluxDB, Prometheus, LogStash)
-* Query & Visualization — that enables visual monitoring and root cause analysis (e.g. Grafana)
+- Collector — daemon(s) to gather metrics such as this repository
+- Time Series database — for storage of real-time, high volume metrics (e.g. InfluxDB, Prometheus, LogStash)
+- Query & Visualization — that enables visual monitoring and root cause analysis (e.g. Grafana)
```
Comment (adriansuarez) [inline README.md:14]: This doesn't change anything in terms of formatting, right?

File: README.md [hunk 3]
```diff
@@ -2,34 +2,35 @@
...
 Built into this container are 4 input plugins to collect metrics from the NuoDB engine:
 
-1.  `metrics` - collects the [Engine Metrics](https://doc.nuodb.com/nuodb/4.0.x/reference-information/metrics-published-by-database-processes/)  on a
+1.  `metrics` - collects the [Engine Metrics](https://doc.nuodb.com/nuodb/4.0.x/reference-information/metrics-published-by-database-processes/) on a
     regular 10s interval.
 2.  `msgtrace` - collects internal NuoDB message tracing data on a regular 30s interval.
 3.  `synctrace` - collects internal NuoDB lock tracing data on a regular 30s interval.
 4.  `threads` - extends the [Telegraf ProcStat Input plugin](https://github.com/influxdata/telegraf/tree/master/plugins/inputs/procstat) with per-thread data.
-Collects host machine resource consumption statistics on a regular 10s interval.
+    Collects host machine resource consumption statistics on a regular 10s interval.
```
Comment (adriansuarez) [inline README.md:26]: What is the extra indent level for?

File: README.md (hunk 2 above)
Comment (NexFlare) [reply README.md:14]: the linting is done Prettier.

**Review by sivanov-nuodb — APPROVED** [review]
Thanks for updating the status badge!

**Review by adriansuarez — APPROVED** [review]

**Pull request merged**